react/features/toolbox/components/ToolbarButton.web.js

```diff
@@ -214,11 +217,11 @@ class ToolbarButton extends AbstractToolbarButton {
 
             if (!button.unclickable) {
                 if (button.tooltipText) {
-                    UIUtil.setTooltipText(this.button,
+                    setTooltipText(this.button,
                         button.tooltipText,
                         tooltipPosition);
                 } else {
-                    UIUtil.setTooltip(this.button,
+                    setTooltip(this.button,
```

Comment:
Why did you move tooltips? Again, I'm not against this but I don't understand what it gives you, especially with reactification of tooltips in review.